Plugins/Creature/Creature.cpp

```diff
@@ -1385,7 +1385,7 @@ ArgumentStack Creature::SetBaseSavingThrow(ArgumentStack&& args)
     if (auto *pCreature = creature(args))
     {
         const auto which = Services::Events::ExtractArgument<int32_t>(args);
-        const auto value = Services::Events::ExtractArgument<int32_t>(args); ASSERT(value >= -128 && value <= 127);
+        const auto value = Services::Events::ExtractArgument<int32_t>(args); ASSERT_OR_THROW(value >= -128 && value <= 127);
```

Comment:
It's generally better if we split and `&&` conditions into two separate asserts. That way when it fires, you know which one it was.